### PR TITLE
feat(ci): Add Tag Release

### PR DESCRIPTION
**Pull request by LucasHartridge**

### CODE REVIEW

**Review by jcunhafonte — CHANGES_REQUESTED** [review]

File: .github/workflows/release-tag.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Push Tag to NPM
```
Comment (jcunhafonte) [inline .github/workflows/release-tag.yaml:1]: Let's improve these names. Also the file name.

File: .github/workflows/release-tag.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+on:
+  push:
+    tags:
+      - 'v*'
```
Comment (jcunhafonte) [inline .github/workflows/release-tag.yaml:6]: This too.

**Review by jcunhafonte — CHANGES_REQUESTED** [review]

File: .github/workflows/release-tag.yaml (hunk 1 above)
Comment (LucasHartridge) [reply .github/workflows/release-tag.yaml]: Updated

File: .github/workflows/release-tag.yaml (hunk 2 above)
Comment (LucasHartridge) [reply .github/workflows/release-tag.yaml]: Updated and configure environment

**Pull request merged**